realm/realm-library/src/main/java/io/realm/internal/SharedGroup.java

```diff
@@ -18,12 +18,21 @@
 
 import java.io.Closeable;
 import java.io.IOError;
+import java.util.concurrent.TimeUnit;
 
+import io.realm.exceptions.IncompatibleLockFileException;
+import io.realm.exceptions.RealmError;
 import io.realm.exceptions.RealmIOException;
 import io.realm.internal.async.BadVersionException;
+import io.realm.internal.log.RealmLog;
 
 public class SharedGroup implements Closeable {
 
+    // Keep these public so we can ask users to experiment with these values if needed.
+    // Should be locked down as soon as possible.
+    public static long[] INCREMENTAL_BACKOFF_MS = new long[] {1, 3, 10, 20}; // Will keep re-using last value until LIMIT is hit
```

Comment:
I'd put a bit more user friendly description on it with date so when users find out, they can see it can be changed, and has a limited lifespan for exposure.

e.g.
>  The timeout and retry-policy settings for opening Realm lock is currently public for now, which will allow users to modify without us needing to release a new version of Realm Java. (June 16, 2016).